premake5.lua

```
@@ -0,0 +1,139 @@
+function getExampleFiles()
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'getExampleFiles'

(W111)